EngineLayer/GlobalVariables.cs

```diff
@@ -202,6 +206,14 @@ public static void AddMods(IEnumerable<Modification> modifications, bool modsAre
             }
         }
 
+        public static void AddSeparationTypes(List<string> separationTypes)
+        {
+            foreach (string separationType in separationTypes)
+            {
+                _SeparationTypes.Add(separationType);
```

Comment:
Instead of iterating through the enumerable, you can just use a single line for this method: "_SeparationTypes.AddRange(separationTypes);"